### ISSUE TITLE
Version 0.83.5 fails on Apple M1 chip

### ISSUE THREAD

**Issue by KevinSchwerdtfeger** [batect/batect#1474]
### Describe the bug

Version 0.83.5 of batect fails with the following error on Apple M1 Max chip

```
➜  batect-0.83.5 ./batect test
minpc= 0x159d00950 min= 0x159d00950 maxpc= 0x15a45c180 max= 0x15a452b50
fatal error: minpc or maxpc invalid
runtime: panic before malloc heap initialized

runtime stack:
runtime.throw({0x15a46dbaf?, 0x0?})
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/panic.go:1047 +0x40 fp=0x16edb2e60 sp=0x16edb2e30 pc=0x159d35b30
runtime.moduledataverify1(0x15ace9120)
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/symtab.go:636 +0x41c fp=0x16edb2f40 sp=0x16edb2e60 pc=0x159d52f3c
runtime.moduledataverify(...)
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/symtab.go:593
runtime.schedinit()
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/proc.go:691 +0x40 fp=0x16edb2fa0 sp=0x16edb2f40 pc=0x159d391f0
runtime.rt0_go()
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/asm_arm64.s:86 +0xd0 fp=0x16edb2fd0 sp=0x16edb2fa0 pc=0x159d641d0
```

Docker version
```
➜  batect-0.83.5 docker version
Client:
 Cloud integration: v1.0.31
 Version:           20.10.23
 API version:       1.41
 Go version:        go1.18.10
 Git commit:        7155243
 Built:             Thu Jan 19 17:35:19 2023
 OS/Arch:           darwin/arm64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.17.0 (99724)
 Engine:
  Version:          20.10.23
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.10
  Git commit:       6051f14
  Built:            Thu Jan 19 17:31:28 2023
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.18
  GitCommit:        2456e983eb9e37e47538f59ea18f2043c9a73640
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### Steps to reproduce

1. Install verion 0.83.5 of batect
2. Create a simple batect.yml file
```
containers:
  test-container:
    image: alpine:3.11.3

tasks:
  test:
    description: Run a test
    run:
      container: test-container
      command: echo 'Hello world!'
```
3. Execute the task `./batect test`

### Expected behaviour

The task is run successfully as it does using batect 0.83.4
```
➜  batect-0.83.4 ./batect test
Version 0.83.5 of Batect is now available (you have 0.83.4).
To upgrade to the latest version, run './batect --upgrade'.
For more information, visit https://github.com/batect/batect/releases/tag/0.83.5.

Running test...
test-container: running echo 'Hello world!'

Hello world!

test finished with exit code 0 in 0.5s.
```

### Actual behaviour

```
➜  batect-0.83.5 ./batect test
minpc= 0x152dfc950 min= 0x152dfc950 maxpc= 0x153558180 max= 0x15354eb50
fatal error: minpc or maxpc invalid
runtime: panic before malloc heap initialized

runtime stack:
runtime.throw({0x153569baf?, 0x0?})
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/panic.go:1047 +0x40 fp=0x17285ee60 sp=0x17285ee30 pc=0x152e31b30
runtime.moduledataverify1(0x153de5120)
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/symtab.go:636 +0x41c fp=0x17285ef40 sp=0x17285ee60 pc=0x152e4ef3c
runtime.moduledataverify(...)
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/symtab.go:593
runtime.schedinit()
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/proc.go:691 +0x40 fp=0x17285efa0 sp=0x17285ef40 pc=0x152e351f0
runtime.rt0_go()
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/asm_arm64.s:86 +0xd0 fp=0x17285efd0 sp=0x17285efa0 pc=0x152e601d0
```

### Version information

```plain text
Unable to obtain version information as this command fails with the same error as running any other task.


➜  batect-0.83.5 ./batect --version
minpc= 0x14d8fc950 min= 0x14d8fc950 maxpc= 0x14e058180 max= 0x14e04eb50
fatal error: minpc or maxpc invalid
runtime: panic before malloc heap initialized

runtime stack:
runtime.throw({0x14e069baf?, 0x0?})
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/panic.go:1047 +0x40 fp=0x16efd6e60 sp=0x16efd6e30 pc=0x14d931b30
runtime.moduledataverify1(0x14e8e5120)
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/symtab.go:636 +0x41c fp=0x16efd6f40 sp=0x16efd6e60 pc=0x14d94ef3c
runtime.moduledataverify(...)
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/symtab.go:593
runtime.schedinit()
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/proc.go:691 +0x40 fp=0x16efd6fa0 sp=0x16efd6f40 pc=0x14d9351f0
runtime.rt0_go()
	/Users/runner/work/docker-client/docker-client/golang-wrapper/build/tools/golang-1.19.5/src/runtime/asm_arm64.s:86 +0xd0 fp=0x16efd6fd0 sp=0x16efd6fa0 pc=0x14d9601d0
```
```


### Any other information

_No response_

**Comment by charleskorn** [comment]
Could you run `./batect --version` with Batect 0.83.4 and share the output here? That will give me some information on your environment that might help track down the issue.

**Comment by KevinSchwerdtfeger** [comment]
As requested, here is the output of `./batect --version` for 0.83.4

```
Batect version:    0.83.4
Built:             2023-01-29 10:56:03 +0000
Built from commit: 87143d15413a6fe93fe5f2f4887da20f726b249f (commit date: 2023-01-29 21:18:22 +1100)
JVM version:       Homebrew OpenJDK 64-Bit Server VM 19.0.2
OS version:        Mac OS X 12.6.1 aarch64
Docker version:    20.10.23 (API version: 1.41, minimum supported API version: 1.12, commit: 6051f14, operating system: 'linux', architecture: 'arm64', experimental: false)
Git version:       2.37.1 (Apple Git-137.1)
```

**Comment by charleskorn** [comment]
Thanks for the information @KevinSchwerdtfeger. I've fixed the issue and I'll release it as v0.83.6 shortly.

**Comment by balasurajgajula** [comment]
Hi @charleskorn 
Facing this issue again in `0.83.8` version

Here, is the version info on `0.83.7` version 👇 
```
> ./batect --version

Batect version:    0.83.7
 Built:             2023-05-19 01:27:00 +0000
 Built from commit: 18d7a7e53ec17051febde6b760d4ed32e17d7a09 (commit date: 2023-05-19 11:08:39 +1000)
 JVM version:       Amazon.com Inc. OpenJDK 64-Bit Server VM 20.0.1
 OS version:        Mac OS X 13.4 aarch64
 Docker version:    24.0.2 (API version: 1.43, minimum supported API version: 1.12, commit: 659604f, operating system: 'linux', architecture: 'arm64', experimental: false)
 Git version:       2.41.0
 
 For documentation and further information on Batect, visit https://github.com/batect/batect.
```

Docker version 👇 
```
> docker version

 Client:
  Cloud integration: v1.0.33
  Version:           24.0.2
  API version:       1.43
  Go version:        go1.20.4
  Git commit:        cb74dfc
  Built:             Thu May 25 21:51:16 2023
  OS/Arch:           darwin/arm64
  Context:           desktop-linux
 
 Server: Docker Desktop 4.20.1 (110738)
  Engine:
   Version:          24.0.2
   API version:      1.43 (minimum version 1.12)
   Go version:       go1.20.4
   Git commit:       659604f
   Built:            Thu May 25 21:50:59 2023
   OS/Arch:          linux/arm64
   Experimental:     false
  containerd:
   Version:          1.6.21
   GitCommit:        3dce8eb055cbb6872793272b4f20ed16117344f8
  runc:
   Version:          1.1.7
   GitCommit:        v1.1.7-0-g860f061
  docker-init:
   Version:          0.19.0
   GitCommit:        de40ad0
```
